libraries/node-core-library/src/FileSystem.ts

```diff
@@ -729,22 +818,92 @@ export class FileSystem {
   }
 
   /**
-   * Copies a file from one location to another.
+   * Copies a single file from one location to another.
    * By default, destinationPath is overwritten if it already exists.
-   * Behind the scenes it uses `fs.copyFileSync()`.
+   *
+   * @remarks
+   * The `copyFile()` API cannot be used to copy folders.  It copies at most one file.
+   * Use {@link FileSystem.copyFiles} if you need to recursively copy a tree of folders.
+   *
+   * The implementation is based on `copySync()` from the `fs-extra` package.
    */
   public static copyFile(options: IFileSystemCopyFileOptions): void {
+    const existsBehavior: AlreadyExistsBehavior = options.alreadyExistsBehavior !== undefined
```

Comment:
There's a pattern for default options in other functions. Use that here?

Reply:
Oh cool, I wasn't aware of that. Good suggestion